src/helpers.js

```diff
@@ -274,7 +274,7 @@ export async function processESLintMessages(messages, textEditor, showRule) {
      keep doing so in later uses.
      */
     const msgLine = line - 1
-    if (typeof endColumn !== 'undefined' && typeof endLine !== 'undefined') {
+    if (typeof endColumn === 'number' && typeof endLine !== 'number') {
```

Comment:
```suggestion
    if (typeof endColumn === 'number' && typeof endLine === 'number') {
```

Reply:
Woops, nice catch.
I've force pushed so a new commit isn't created just for that fix.